content/sklearn/content/kernel-ridge-regression/kernel-ridge-regression.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Kernel Ridge Regression'
```

Comment:
``` suggestion
Title: 'Kernel Ridge Regression'
```
Title name in docs is always written in lowercase and separated by dashes instead of spaces. Please take a note of it.

Reply:
Do you mean the file name or title name? Not sure why I would need to do that in the title name. This is a concept entry, not a term entry.

Reply:
> Do you mean the file name or title name? Not sure why I would need to do that in the title name. This is a concept entry, not a term entry.

I mean the title name must be the same as the file name and the title name must always be in lowercase and separated by dashes instead of spaces.

Reply:
Please link me where it states that in the Contribution Guide docs. From what is specified in the style-guide.md file, I followed this template:

**Title: 'The Title'** <-- Not lowercase or separated by dashes.
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
...

Can you please briefly list in bullet points what makes this an invalid PR?

Reply:
Pinging here for a response in what makes this PR invalid.